core/src/lib.rs

```diff
@@ -45,18 +46,32 @@ pub enum ValueType {
 
 impl ValueType {
     pub fn to_edn_value(&self) -> edn::Value {
-        match self {
-            &ValueType::Ref => values::DB_TYPE_REF.clone(),
-            &ValueType::Boolean => values::DB_TYPE_BOOLEAN.clone(),
-            &ValueType::Instant => values::DB_TYPE_INSTANT.clone(),
-            &ValueType::Long => values::DB_TYPE_LONG.clone(),
-            &ValueType::Double => values::DB_TYPE_DOUBLE.clone(),
-            &ValueType::String => values::DB_TYPE_STRING.clone(),
-            &ValueType::Keyword => values::DB_TYPE_KEYWORD.clone(),
+        match *self {
```

Comment:
I did this on the way, but I suspect it should actually be `to_edn_value(self)` — `ValueType` is `Copy`, so avoiding references wherever possible is best.